pxtcompiler/emitter/service.ts

```diff
@@ -1158,26 +1163,18 @@ namespace ts.pxtc.service {
                         if (paramType) {
                             // if this is a property access, then weight the results higher if they return the
```

Comment:
should these two lines of comments be dropped? Does not appear to only be applying to property accesses anymore.